gitlab/data_source_gitlab_users.go

```diff
@@ -11,25 +16,62 @@ func dataSourceGitlabUsers() *schema.Resource {
 		Read: dataSourceGitlabUsersRead,
 
 		Schema: map[string]*schema.Schema{
-			"order_by": {
-				Type:     schema.TypeString,
+			"options": {
```

Comment:
Why nest the search options in an `options` map? This could make sense if several `options` blocks could be defined but this isn't the case here.

Reply:
I wanted to avoid changing the parameter "provider". But I agree it makes more sense having all options at the root.